### ISSUE TITLE
Doesn't work in sketch 3

### ISSUE THREAD

**Issue by smlombardi** [ddwht/sketch-dynamic-button#2]
See animated gif below.  Unless I 
![button](https://cloud.githubusercontent.com/assets/835148/2695819/55e5a4be-c3db-11e3-80ef-d9c21073f587.gif)
am using it wrong.

**Comment by sangdth** [comment]
Does not work on Sketch 3.3 :(

**Comment by giuseppebergman** [comment]
Right, on Sketch 3.3 it just renames your text layer to "0:0:0:0" and creates an emtpy "flex button" group at the same level.
HOWEVER, you can rebuild the structure manually, by moving your "0:0:0:0" text layer in the folder, then adding a rectangle named "BG" (case sensitive). From then on, the plugin will work.

**Comment by selimimoberdorf** [comment]
When will be there a Fix for Sketch 3.3? I worked a lot with your Plugin and do it manually is really a pain in the a**, if i have more then 100 Buttons. ;-)

**Comment by ddwht** [comment]
Did you try to download new version? It's already work with 3.3

**Comment by selimimoberdorf** [comment]
Sorry my fault! Sketch Toolbox didn't updated the Plugin…

**Comment by denswor** [comment]
@ddwht i've installed last version but plugin doesn't works. it still just renames your text layer to "0:0:0:0" and creates an emtpy "flex button" group at the same level.
what am i doing wrong?

**Comment by matthewalandaniels** [comment]
Downloaded and installed latest but still not working. I'm using Sketch V 3.3.2.
When I select text and run the plugin, it doesn't create a folder or a background shape. It does change the name of the text layer to 0,0,0,0. It also repositions the layer to the top left of the art-board. 
Do you know when there will be a working version of Dynamic-Button for V 3.3.2?
Thanks!

**Comment by giuseppebergman** [comment]
Works perfectly here, w/ Sketch 3.3.2. 
Also, @matthewalandaniels : I'm not the developer ;)

**Comment by selimimoberdorf** [comment]
@matthewalandaniels It's working. If you use «Sketch Toolbox» you have to remove and install it again. Sometimes it doesn't update it automatically.

**Comment by Gabszanto** [comment]
Hey, it seems like it's not working on version 3.4.2.
The padding moves back to 0:0:0:0 whenever I click on cmd+J for the second time. Any fix?

**Comment by ebdiver** [comment]
I'm experiencing the same issue as Gabszanto.

**Comment by ghost** [comment]
Same error here, I always got a Dynamic group folder with no BG.
![screen shot 2016-01-27 at 8 47 16 pm](https://cloud.githubusercontent.com/assets/16928220/12631231/7917a4b4-c537-11e5-8861-2ada88998b31.png)

**Comment by shoglund2000** [comment]
Yes, I get the exact "Dynamic group" folder with no BG box (using Sketch 3.6.1)

**Comment by JoeyQiang** [comment]
Yes, I get the same issue now. Using Sketch(3.7.2), there is no BG layer after did the cmd + J.

**Comment by ryuujinsilver** [comment]
I also have this issue. Using Sketch Beta Version 3.8 (29681)

**Comment by mistercamacho** [comment]
Use this one:
https://github.com/fuggfuggfugg/sketch-dynamic-button-3.5
